purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -117,9 +132,9 @@ fun Purchases.purchaseProductWith(
     skuDetails: SkuDetails,
     upgradeInfo: UpgradeInfo,
     onError: MakePurchaseErrorFunction = onMakePurchaseErrorStub,
```

Comment:
is the name `MakePurchaseErrorFunction` still accurate? should it be `ProductChangeErrorFunction` or even `PurchaseErrorFunction`?